wgpu/Cargo.toml

```diff
@@ -175,11 +175,7 @@ static_assertions.workspace = true
 ###################
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 # Needed for only wgpu-core backend. Not optional as only wgpu-core backend exists on native platforms.
-wgpu-core = { workspace = true, features = [
-    "raw-window-handle",
-    "renderdoc",
-    "indirect-validation",
-] }
+wgpu-core = { workspace = true, features = ["raw-window-handle", "renderdoc"] }
```

Comment:
Having this be disabled in `wgpu` feels wrong - there definitely should be some way of turning this on

Reply:
Will enable `wgsl` unconditionally on native as discussed.